cmd/kube-controller-manager/app/core.go

```diff
@@ -188,6 +189,21 @@ func startAttachDetachController(ctx ControllerContext) (bool, error) {
 	return true, nil
 }
 
+func startVolumeExpandController(ctx ControllerContext) (bool, error) {
+	expandController, expandControllerErr := expand.NewExpandController(
```

Comment:
Check `if utilfeature.DefaultFeatureGate.Enabled(features.ExpandPersistentVolumes) {` and don't start the controller if the feature is not enabled.